BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -279,8 +279,23 @@ extension ElementContent {
     ///
     /// This is useful if you are placing the element in a nested `BlueprintView`, for example (eg
     /// to create a stateful element) and just need this element to be correctly sized.
-    public init(measuring element: Element) {
-        storage = MeasureElementStorage(child: element)
```

Comment:
Posting here for posterity: We could expose a method like

```swift
extension Element {
   func asSpacer(along axes: MeasuringAxes = .both) { ... } 
}
```

So that you could use any ol' element as a dynamic spacer – which has been an infrequent but not _uncommon_ ask. I think for now, I'm not going to add this, but this would make it easier for some of these cases. I might put a call out in #market-ios to see if anyone would find this helpful.